site/en/tutorials/images/classification.ipynb

```diff
@@ -22,7 +22,7 @@
       "outputs": [],
       "source": [
         "#@title Licensed under the Apache License, Version 2.0 (the \"License\");\n",
-        "# you may not use this file except in compliance with the License.\n",
+        "# You may not use this file except in compliance with the License.\n",
```

Comment:
Please don't change the license. This is how they apply it: https://www.apache.org/licenses/LICENSE-2.0#apply